test/integration/src/peggy2/test_peggy2_wrong_signature.py

```diff
@@ -0,0 +1,30 @@
+import siftool_path
+from siftool import eth, test_utils, sifchain
+from siftool.common import *
+
+
+fund_amount_eth = 10 * eth.ETH
+fund_amount_sif = 10 * test_utils.sifnode_funds_for_transfer_peggy1  # TODO How much rowan do we need? (this is 10**18)
+
+
+def test_sign_prophecy_with_wrong_signature_grpc(ctx):
+    # Create/retrieve a test ethereum account
+    test_eth_account = ctx.create_and_fund_eth_account(fund_amount=fund_amount_eth)
+
+    # create/retrieve a test sifchain account
+    moniker = "temp-moniker"
```

Comment:
actually, the problem is we need the val address instead of acc address. Currently, there is no method to get it or conversion method from acc address to val address. maybe we can implement it in the future.

Reply:
You mean this? https://github.com/Sifchain/sifnode/blob/ec4d14404a51ac6e7ae0c81ecb6f0181e36183a3/test/integration/framework/src/siftool/sifchain.py#L81

Reply:
yes, it need the moniker to get val address.